aliases

```diff
@@ -1,10 +1,3 @@
-e.=explorer .
```

Comment:
Yeah, I'm removing the aliases that are there by default when you download cmder. If someone does a complete copy-paste, and new default aliases added will get lost, and if they don't do a complete copy-paste, there will be duplicates. We should only have custom aliases in this file imo.